changelog/config.js

```diff
@@ -361,7 +360,10 @@ module.exports = {
 				}
 				eosDate.setMonth(eosDate.getMonth() + 6);
 			}
-			context.eosDate = dateFormat(eosDate, 'yyyy-mm-dd', true);
+			const yyyy = eosDate.getUTCFullYear();
+			const mm = (eosDate.getUTCMonth() + 1).toString().padStart(2, '0');
+			const dd = eosDate.getUTCDate().toString().padStart(2, '0');
+			context.eosDate = `${yyyy}-${mm}-${dd}}`;
```

Comment:
I see a trailing curly brace that we probably don't want in the date string.

Furthermore, why not just make this whole thing simpler and do:

```js
eosDate.toISOString().split('T')[0]
```

Reply:
Fixed. `eosDate.toISOString().split('T')[0]` looks much better.